### ISSUE TITLE
File is damaged or corrupt

### ISSUE THREAD

**Issue by Deson-Maquilang-MYT** [nigification/Flowgorithm-Mac-OSX#30]

**Comment by kwikysw** [comment]
still the same btw

**Comment by cryptobyls** [comment]
I just used the fix he proposed in another thread and it worked fine for me with the V.4 beta

Copy/paste in the terminal:
> xattr -d com.apple.quarantine /Applications/Flowgorithm.app

**Comment by citizengordon** [comment]
Came here with the same issue: Macbook Pro (Intel i9), Sequoia 15.2 

Opening the app got the "file damaged, move to Trash" popup, and the app would not open.

I am installing the latest stable version, 3.4.2

Running the command mentioned above:

> xattr -d com.apple.quarantine /Applications/Flowgorithm.app 

corrects (or masks?) the issue and I am able to open the app.

**Comment by jostasik** [comment]
Yes, xattr -d com.apple.quarantine /Applications/Flowgorithm.app is how you fix the issue. 

This error arises from the app being distributed from an "Unidentified Developer" (a.k.a. Me) in zip file, instead of the app store.
